docs/reference/async-processing.mdx

```diff
@@ -0,0 +1,116 @@
+---
+title: Async Processing
+---
+import {Icons} from '/src/components/icons'
+
+:::info
+
+This page covers the Responsive Async Processor: its benefits, its limitations, how to use it, and how to tune it.
+
+The Async Processor is available with Responsive client **v0.24.0** onward.
+:::
+
+## Why use the Async Processor
+Async processing allows Kafka Streams applications to concurrently process records within a single task, thus changing the upper limit of parallelization from the level of a Kafka partition to the level of a record key. 
+
+It works by giving each StreamThread a dedicated thread pool which it uses to process the records passing through an async processor. Records of the same key are still processed in order and all the same consistency and correctness guarantees of Kafka Streams are maintained, including exactly-once semantics.
+
+While async processing can help improve performance of any application with a large keyspace, it was built to shine when processors include a long-running step such as remote state store requests or generic RPCs. The classic solution for achieving high throughput when you are bottlenecked by network latency is to increase the parallelism of the calls to the target system, which is exactly what the async processor does for Kafka Streams. 
+
+This makes the Responsive Async Processor the perfect tool to complement Responsive state stores, meaning you don’t need to sacrifice performance in order to gain the availability and elasticity benefits of remote storage with Kafka Streams!
+
+## Expected Performance Gains
+
+As with all real-world systems, performance depends heavily on the specific use case and parameters. However, in all of our experiments, async processing showed an improvement over regular Kafka Streams when a topology includes a processor which makes a remote call. 
+
+<img
+    alt={"Responsive Asyc Processor Peformancs"}
+    src={require('/static/img/docs/reference/async-processing/async-processing-perf.png').default}
+    className={'px-[10%] mb-4'} />
+
+
+To give an idea of the benefits of using the async processor when your topology includes higher latency external calls, we ran a test that injected a variable latency in our remote storage client while running a simple application that would deduplicate events over a window. A couple of things stand out from the results above: 
+
+1. The greater the remote latency, the bigger the impact of async. At 2 ms round trip latency, throughput improves by 5.5x. At 1 ms round trip latency, throughput improves by 3x.
```

Comment:
One nit about the graphic and how we talk about it here -- it feels a little weird to have 2ms first and 1ms last. I know it makes sense for us since the lower latency actually means higher throughput, but it just looks a little weird to see the x axis moving backwards on the graph. took me an extra second to understand what I was looking at.

not a huge deal though, your call, just wanted to mention it since if it bothers/confused, me it'll probably bother other people too 🤷‍♀️

Reply:
I think you are right. I went back and forth over it. I think it's more conventional to have the x-axis be ascending. Will update.